Parsers/ASimAuditEvent/Tests/VMwareCarbonBlackCloud_AsimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,90 @@
+﻿Result
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcHostname]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [ObjectId]"
+"(1) Warning: Missing recommended field [TargetHostname]"
+"(1) Warning: Missing recommended field [TargetIpAddr]"
```

Comment:
can we map missing recommended fields?

Reply:
Dvc and Target would be same EDR case

Reply:
As of now we are getting single IP field which of where the events get generated that is why we have mapped that ip to Dvc and not to the Target.